atef/tests/conftest.py

```diff
@@ -118,6 +119,15 @@ def qapp(pytestconfig):
     return application
 
 
+@pytest.fixture(scope='session', autouse=True)
+def ophyd_cleanup():
+    """Clean up ophyd - avoid teardown errors by stopping callbacks."""
+    yield
+    dispatcher = ophyd.cl.get_dispatcher()
```

Comment:
Nitpick - if this ends up working - could we reuse `atef.utils.ophyd_cleanup()` instead?

Also, one more thing - in `atef.tests.test_commandline`, I think we should pass `cleanup=False` to `main()`...